public/locales/en/translation.json

```diff
@@ -1,4 +1,28 @@
 {
+  "contact": {
+    "contactUs":{
+      "message":["Follow Us",
+      "on social media"],
+      "form":["Contact Us",
```

Comment:
I know I could do them like nameField, surnameField bla etc. but i didnt want to be keys to be that long so divided them to logical arrays

Reply:
you should be doing it that way, the keys being longer is more important than having an array of 10

imagine you coming back to this code in 3 weeks
or imagine if someone besides you was translating (usually translation work is outsourced). 

please do the same for info

you should almost never be using arrays in general for translations like this

Reply:
done @louisrli